cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -183,99 +183,7 @@ Slice::CsGenerator::fixId(const string& name, unsigned int baseTypes, bool mangl
 string
 Slice::CsGenerator::getOptionalFormat(const TypePtr& type)
 {
```

Comment:
The code coverage report marked `Slice::Type::getOptionalFormat` as unused.
But instead of deleting it, the correct fix was to start using it.

Every single language has it's own `getOptionalFormat` which is just a re-implementation of the original one, with some prefix slapped on front. So I deleted all the redundant code, and now they all just call the `getOptionalFormat` defined in the parser...